docs/components/modeler/desktop-modeler/process-applications.md

```diff
@@ -70,3 +72,15 @@ Any file within a process application can be linked as a resource. Linking a res
 - Manually by setting the process, decision, or form ID in the properties panel
 
 <p><img src={LinkResourcesImg} alt="Linking resources by using the replace feature" /></p>
+
+## Deploying a process application
+
+Process applications can be deployed using the [deploy feature](./connect-to-camunda-8.md). When deploying a process application, all files that are part of the process application will be deployed.
+
+<p><img src={DeployImg} alt="Deploying a process application" /></p>
+
+## Starting an instance of the main process
+
+To start an instance of the main process of a process application use the [start instance feature](./start-instance.md). Before starting an instance of the main process all files that are part of the process application will be deployed.
```

Comment:
```suggestion
To start an instance of any process inside of the application use the [start instance feature](./start-instance.md). Under the hood, this will ensure that the application is deployed to your configured Camunda 8 cluster.
```

Reply:
I simplifyied it removing the notion of a main process. I had added it initially because that's probably the happy path.